src/liballoc/collections/vec_deque.rs

```diff
@@ -202,6 +202,22 @@ impl<T> VecDeque<T> {
                                  len);
     }
 
+    /// Copies all values from `src` to `self`, wrapping around if needed.
+    /// Assumes capacity is sufficient.
```

Comment:
This comment should be expanded to explain where in the `VecDeque` is the slice copied.